system/cache/providers/CFProvider.cfc

```diff
@@ -493,7 +494,9 @@ component
 	 * @objectKey The object cache key
 	 */
 	boolean function clearQuiet( required objectKey ){
-		if( server.coldfusion.productVersion.listFirst() == 2018 ){
+		var ACFModernEngines = [ 2016, 2018, 2021 ];
```

Comment:
2016 shouldn't be added here in all reality, only 2018 plus.
Also, why not just do a gt 2018 for the version?